pkgs/development/python-modules/pytorch/binary-hashes.nix

```diff
@@ -48,4 +48,51 @@ version : builtins.getAttr version {
       hash = "sha256-7qZ17AHsS0oGVf0phPFmpco7kz2uatTrTlLrpwJtwXY=";
     };
   };
```

Comment:
I kept the old pytorch 1.10 binary hashes around, but I'm happy to delete them if that is the correct workflow.

Reply:
I also think you can delete previous pytorch 1.10.0 binary hashes.